CONTRIBUTING.md

```diff
@@ -135,31 +147,37 @@ encrypted databases. More exactly, `CDTDatastore` relies on [FMDB][FMDB] to
 access SQLite databases and FMDB is able to do that using the standard iOS
 library for SQLite or SQLCipher.
 
-If SQLCipher is not included but you try to create a `CDTDatastore` with a
-encryption key (providing an instance that conforms to protocol
-[CDTEncryptionKeyProvider][CDTEncryptionKeyProvider]), the operation will fail
-given that we lack the code to cipher the database. But it will succeed if the
-pod for SQLCipher is in the workspace.
+If SQLCipher is not included but a `CDTDatastore` is created with an
+encryption key (an instance that conforms to protocol
+[CDTEncryptionKeyProvider][CDTEncryptionKeyProvider]), the operation
+will fail.
+
+The `CDTDatastore` and `CDTDatastoreOSX` targets can be used to build
+the library with SQLCipher support.
 
-This is the reason why we need two different workspaces. The code does not
-change, however the behaviour is different depending of the libraries included.
-The `Podfile` in `EncryptionTests` is configured to import FMDB with SQLCipher
-while `CDTDatastore.xcworkspace` uses the default version of FMDB.
-`EncryptionTests.xcworkspace` has a sub-set of the tests in
-`CDTDatastore.xcworkspace`, the tests in the former are expected to succeed
-when a key is provided while the tests in the second have to fail.
+The `CDTDatastoreEncryptionTests` and `CDTDatastoreEncryptionTestsOSX`
+targets will build and run the sub-set of tests which take a key and
+exercise the database encryption functionality.
 
-If you want CDTDatastore to cipher the databases, you do not need to include
-SQLCipher as a pod in your `Podfiles`, replace:
+If you want CDTDatastore to cipher the databases, you do need to include
+the `Encryption` variant as a pod in your `Podfiles`, replace:
 
 ```
 pod "CDTDatastore"
 ```
+or
+```
+pod "CDTDatastoreOSX"
```

Comment:
There isn't a pod called `CDTDatastoreOSX`, thats a target in the project, the Pod is `CDTDatastore`. I'm not sure why we have this section on the contributing, perhaps we should drop everything in this section from line 161 down